dev-python/poetry/poetry-1.6.1-r1.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=poetry
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A frontend for poetry - a python dependency management and packaging tool"
+HOMEPAGE="
+	https://python-poetry.org/
+	https://github.com/python-poetry/poetry
+	https://pypi.org/project/poetry/
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/poetry-core-1.7.0[${PYTHON_USEDEP}]
+	>=dev-python/poetry-plugin-export-1.5.0[${PYTHON_USEDEP}]
+	>=dev-python/build-0.10.0[${PYTHON_USEDEP}]
+	!>=dev-python/build-1.0.0[${PYTHON_USEDEP}]
```

Comment:
Please don't use blockers for this - they're only for use with file collisions.

You need to either:
a) just < dep
or
b) just >= and <